iree/turbine/kernel/boo/ops/layout_customizable_conv.py

```diff
@@ -8,31 +8,103 @@
 
 import torch
 
+from .library import register_meta, BOO_LIBRARY
+from .utils import *
+
 from ..conv_exports import (
     ConvSignature,
     DEFAULT_LAYOUTS,
     Permutation,
 )
-from ..driver.launch import get_launchable
 
+from ..driver.launch import get_launchable
 from ..runtime import LaunchableRuntimeCache
 
-from .library import define_schema, register_impl, register_meta
-from .utils import *
+from ....runtime.op_reg import CustomOp, KernelBuilder, KernelSelection, impl_helper
+from ....transforms.merger import Merger
+from ....support.ir_imports import Operation, StringAttr
 
 __all__ = [
     "boo_layout_customizable_convolution",
+    "convolution_replacement",
 ]
 
 # Forward Convolution Implementations #
 
-define_schema(
-    "layout_customizable_convolution",
-    "(Tensor x, Tensor w, Tensor? b, int[] stride, int[] padding, int[] dilation, int groups, str input_layout, str kernel_layout, str output_layout) -> Tensor",
-)
+
+@CustomOp.register(library=BOO_LIBRARY)
+class layout_customizable_convolution(CustomOp):
+    signature = "layout_customizable_convolution(Tensor x, Tensor w, Tensor? b, int[] stride, int[] padding, int[] dilation, int groups, str input_layout, str kernel_layout, str output_layout) -> Tensor"
```

Comment:
can we use the full names everywhere
```suggestion
    signature = "layout_customizable_convolution(Tensor input, Tensor weight, Tensor? bias, int[] stride, int[] padding, int[] dilation, int groups, str input_layout, str kernel_layout, str output_layout) -> Tensor"
```